app.js

```diff
@@ -104,19 +92,38 @@ class ToDoList {
         for (let i = 0; i < taskList.length; i++) {
             if (taskList[i].id == idOfTaskToChange) {
                 taskList[i].completed = e.target.checked;
+                this.setLocalStorage();
                 break;
             }
         }
     }
 
-    render = () => {
-        const taskList = this.toDoTasks;
+    searchTasks = () => {
+        const searchText = searchInput.value.toLowerCase();
+        let tasks = this.toDoTasks;
+
+        tasks = tasks.filter(el => el.task.toLowerCase().includes(searchText));
+        this.taskSection.textContent = '';
+
+        this.render(tasks);
+    }
 
+    setLocalStorage = () => {
+        localStorage.setItem('toDoTasks', JSON.stringify(this.toDoTasks));
+    }
+
+    render = (taskList = this.toDoTasks) => {
```

Comment:
Czemu to przywróciłeś?

Reply:
`searchTasks = () => {
        const searchText = searchInput.value.toLowerCase();
        let tasks = this.toDoTasks;

        tasks = tasks.filter(el => el.task.toLowerCase().includes(searchText));
        this.taskSection.textContent = '';

        **this.render(tasks);**
    }`

tutaj tego użyłem

Reply:
`    searchTasks = () => {
        const searchText = searchInput.value.toLowerCase();
        let tasks = this.toDoTasks;

        tasks = tasks.filter(el => el.task.toLowerCase().includes(searchText));
        this.taskSection.textContent = '';

        this.render(tasks);
    }`

tutaj tego użyłem

Reply:
Yup, cofam to :)